src/encoded/schemas/page.json

```diff
@@ -70,7 +70,7 @@
                 "deleted",
                 "inactive",
                 "in review",
-                "public",
```

Comment:
We currently have `public` pages, so that might leaf to problems. Add `open` instead?

Reply:
Oh, you are changing the status in the master-inserts. But if there are pages created via the portal, that might still lead to issues?